integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorUpgrade.java

```diff
@@ -102,6 +99,19 @@ public static void init() {
 
   }
 
+  /**
```

Comment:
We do not need upgrade from Release 2.5.0. This test case should be removed.

Reply:
yes, remove it ..

Reply:
it's removed in develop branch. I just synced with it